### PR TITLE
Bootstrap sdk net for machinelearningservices

### PR DESCRIPTION
**Pull request by harneetvirk**

### CODE REVIEW

File: sdk/machinelearningservices/Azure.ResourceManager.MachineLearningServices/src/autorest.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+
+azure-arm: true
+require: https://raw.githubusercontent.com/forteddyt/azure-rest-api-specs/db218390cc57f2509db0e88d8d5ffd1872aa9435/specification/machinelearningservices/resource-manager/readme.md
```
Comment (christothes) [inline sdk/machinelearningservices/Azure.ResourceManager.MachineLearningServices/src/autorest.md:8]: Is this intentionally not in master? I believe you will encounter issues with auto-generation if it is in a feature branch.

File: sdk/machinelearningservices/Azure.ResourceManager.MachineLearningServices/src/properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+using System.Runtime.CompilerServices;
+
+// Replace with test project/test project public key and uncomment to make internal members visible to
+// your test project. If not needed, this can be deleted.
+// [assembly: InternalsVisibleTo("Azure.Template.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100d15ddcb29688295338af4b7686603fe614abd555e09efba8fb88ee09e1f7b1ccaeed2e8f823fa9eef3fdd60217fc012ea67d2479751a0b8c087a4185541b851bd8b16f8d91b840e51b1cb0ba6fe647997e57429265e85ef62d565db50a69ae1647d54d7bd855e4db3d8a91510e5bcbd0edfbbecaa20a7bd9ae74593daa7b11b4")]
+
+// Replace Microsoft.Test with the correct resource provider namepace for your service and uncomment.
+// See https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-services-resource-providers
+// for the list of possible values.
+// [assembly: Azure.Core.AzureResourceProviderNamespace("Microsoft.Test")]
```
Comment (christothes) [inline sdk/machinelearningservices/Azure.ResourceManager.MachineLearningServices/src/properties/AssemblyInfo.cs:13]: Is this neeeded?

File: sdk/machinelearningservices/Azure.ResourceManager.MachineLearningServices/src/autorest.md (hunk 1 above)
Comment (harneetvirk) [reply sdk/machinelearningservices/Azure.ResourceManager.MachineLearningServices/src/autorest.md:8]: This is intentional change from a feature branch. The swaggers from spec repo are not merged to master yet.

File: sdk/machinelearningservices/Azure.ResourceManager.MachineLearningServices/src/properties/AssemblyInfo.cs (hunk 2 above)
Comment (harneetvirk) [reply sdk/machinelearningservices/Azure.ResourceManager.MachineLearningServices/src/properties/AssemblyInfo.cs:13]: Not required. Removing it on next iteration.

**Review by christothes — APPROVED** [review]

**Pull request merged**